solidity/contracts/bridge/BridgeState.sol

```diff
@@ -162,68 +143,31 @@ library BridgeState {
         // This is a per-redemption output max fee for the redemption
         // transaction.
         uint64 redemptionTxMaxFee;
+        // slither-disable-next-line unused-state
+        bytes8 __redemptionAlignmentGap;
```

Comment:
Could we add a comment explaining why we added this variable, something like:
// for a more efficient variable layout